File: app/src/main/java/com/fengniao/news/bean/ZhiHuArticleDetail.java
package com.fengniao.news.bean;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ZhiHuArticleDetail {
    public String body;
    public String title;
    public String image;
    public int id;
    public Section section;
    @SerializedName("css")
    public List<String> cssList;
    public List<Recommenders> recommenders;

    private class Section {
        public String thumbnail;
        public int id;
        public String name;
    }

    private class Recommenders {
        public String avatar;
    }
}

File: app/src/main/java/com/fengniao/news/bean/ZhiHuArticleDetail.kt
package com.fengniao.news.bean

import com.google.gson.annotations.SerializedName

class ZhiHuArticleDetail {
    var body: String? = null
    var title: String? = null
    var image: String? = null
    var id: Int = 0
    var section: Section? = null
    @SerializedName("css")
    var cssList: List<String>? = null
    var recommenders: List<Recommenders>? = null

    inner class Section {
        var thumbnail: String? = null
        var id: Int = 0
        var name: String? = null
    }

    inner class Recommenders {
        var avatar: String? = null
    }
}
